xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetabledate.js

```diff
@@ -63,6 +63,21 @@ require(['jquery', 'jdateformatparser', 'daterangepicker', 'xwiki-events-bridge'
           updateInput(this, event, picker);
         });
 
+        
+        input.on('keypress', function (e) {
+          if(e.which !== 13) return;
+          var txt = input.val().trim();
+          var range = txt.split(' - ');
+          if (range.length == 2){
```

Comment:
You can use the triple equals operator as an optimization (especially since I've seen you used the negation ``!==`` above). Also, a space is needed before ``{`` in order to follow the code style.